### PR TITLE
EID-1105: Upgrade to Java 11.

### PR DESCRIPTION
**Pull request by tomrosier**
Upgrade the Gradle from 4.10 -> 4.10.2
Migrate from the deprecated `sun.security.pkcs11.SunPKCS11` to using `Security.getProvider("SunPKCS11").getClass()`.

### CODE REVIEW

File: src/main/java/uk/gov/ida/eidas/cli/metadata/SignWithSmartcard.java [hunk 1]
```diff
@@ -25,7 +26,7 @@
 
   @Override
   public Void call() throws Exception {
-    PKCS11KeyLoader keyLoader = new PKCS11KeyLoader(sun.security.pkcs11.SunPKCS11.class, pkcs11Config, password);
+    PKCS11KeyLoader keyLoader = new PKCS11KeyLoader(Security.getProvider("SunPKCS11").getClass(), pkcs11Config, password);
```
Comment (JHjava) [inline src/main/java/uk/gov/ida/eidas/cli/metadata/SignWithSmartcard.java]: Do we need any other additional config for this to work?

Comment (minhngocd) [reply src/main/java/uk/gov/ida/eidas/cli/metadata/SignWithSmartcard.java]: you can test it out with the local jar to see it work.

@tomrosier nice avatar 👌

Comment (tomrosier) [reply src/main/java/uk/gov/ida/eidas/cli/metadata/SignWithSmartcard.java]: It just seems to wrap `sun.security.pkcs11.SunPKCS11` and make it accessible through the 'standard Java api'. Will local test it now 😄

Comment (tomrosier) [reply src/main/java/uk/gov/ida/eidas/cli/metadata/SignWithSmartcard.java]: Also Cheers Mino! @minhngocd

**Comment by tomrosier** [comment]
Can someone test this with a local smart card (Yubikey) before this is merged.

**Pull request closed without merging**